<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindows</class>
 <widget class="QMainWindow" name="MainWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Redes de Sensor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Salir">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(170, 170, 170);</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagenes/casa.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="Radio_Bt_1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rad_Bt_1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Radio_Bt_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rad_Bt_2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 158, 158);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bt_1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bt_2</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Perros</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Video Juegos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gatos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Motos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Carros</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flores</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cascos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Controles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Electronica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dibujos</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionClose_File"/>
   </widget>
   <widget class="QMenu" name="menuEdict">
    <property name="title">
     <string>Edict</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuForm">
    <property name="title">
     <string>Form</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdict"/>
   <addaction name="menuForm"/>
   <addaction name="menuView"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="text">
    <string>Close File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindows</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
